examples/repl.rs

```diff
@@ -30,6 +30,14 @@ mod example {
             }
         }
 
+        fn mut_remote(&mut self) -> Result<&mut LinuxTarget, Box<dyn std::error::Error>> {
```

Comment:
nit: Please take this with a grain of salt as I don't know what I am talking, but it seems to be the convention to use `as_mut` as the name ?

Reply:
That would be a good name if `Self` were to be an owned version of the returned type. For example `Vec::as_mut` -> `&mut [u8]`. In this case `LinuxTarget` is just one of the fields.

Reply:
makes sense